core/modules/qmeta/QStatusMysql.cc

```diff
@@ -0,0 +1,189 @@
+/*
+ * LSST Data Management System
+ * Copyright 2019 AURA/LSST.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <https://www.lsstcorp.org/LegalNotices/>.
+ */
+
+// Class header
+#include "qmeta/QStatusMysql.h"
+
+// System headers
+#include <algorithm>
+
+// Third-party headers
+#include "boost/lexical_cast.hpp"
+#include <boost/algorithm/string/replace.hpp>
+
+// LSST headers
+#include "lsst/log/Log.h"
+
+// Qserv headers
+#include "qmeta/Exceptions.h"
+#include "qmeta/QMetaTransaction.h"
+#include "sql/SqlResults.h"
+
+
+namespace {
+
+// Current version of QStatus schema.
+char const VERSION_STR[] = "0";
```

Comment:
I don't see any uses for this variable in the rest of the code. Besides, we already have a schema versioning mechanism implemented in table  `qservMeta.QMetadata`:
```
MariaDB [qservMeta]> SELECT * FROM qservMeta.QMetadata;
+---------+-------+
| metakey | value |
+---------+-------+
| version | 1     |
+---------+-------+
```